### ISSUE TITLE
Chokes on null coalescing assignment on PHP 7.2, 7.3 from vendor

### ISSUE THREAD

**Issue by localheinz** [vimeo/psalm#5044]
Not sure if this is a problem with `vimeo/psalm`, but I ran into a problem running `vimeo/psalm` in a project that indirectly requires [`symfony/polyfill-mbstring:1.22.0`](https://github.com/symfony/polyfill-mbstring/tree/v1.22.0).

### Versions

- `vimeo/psalm:3.18.2`
- PHP 7.2.34
- PHP 7.3.25

## Steps required to reproduce

```
$ git clone git@github.com:ergebnis/php-cs-fixer-config.git
$ cd php-cs-fixer-config
$ composer install
```

Then run

```
$ vendor/bin/psalm --config=psalm.xml --diff --show-info=false --stats --threads=4
```

### PHP 7.2, 7.3, 7.4

No errors.

### PHP 7.2, 7.3

```
Scanning files...
Analyzing files...

░░░░░░░░░░░░░░░░░░░░░

ERROR: ParseError - vendor/symfony/polyfill-mbstring/bootstrap80.php:125:86 - Syntax error, unexpected '=' on line 125 (see https://psalm.dev/173)
    function mb_scrub(string $string, string $encoding = null): string { $encoding ??= mb_internal_encoding(); return mb_convert_encoding($string, $encoding, $encoding); }


------------------------------
1 errors found
------------------------------
5 other issues found.
You can display them with --show-info=true
------------------------------

Checks took 4.26 seconds and used 195.981MB of memory
Psalm was able to infer types for 100% of the codebase
-----------------
100% src/Factory.php (0 mixed)
100% src/RuleSet/AbstractRuleSet.php (0 mixed)
100% test/AutoReview/SrcCodeTest.php (0 mixed)
100% test/Double/Config/RuleSet/DummyRuleSet.php (0 mixed)
100% test/Unit/FactoryTest.php (0 mixed)
100% test/Unit/RuleSet/AbstractRuleSetTestCase.php (0 mixed)
100% test/Unit/RuleSet/ExplicitRuleSetTestCase.php (0 mixed)
```

:bulb: See https://github.com/symfony/polyfill-mbstring/blob/v1.22.0/bootstrap80.php#L125.

### PHP 7.4

No errors.

**Comment by localheinz** [comment]
@psalm-github-bot

Sorry, no!

**Comment by localheinz** [comment]
Solved by updating `vimeo/psalm`. 

Sorry to waste your time!

**Comment by weirdan** [comment]
Yeah, fixed in #4967 (released in 4.4.0).

**Comment by localheinz** [comment]
Thank you, @weirdan!
